statistics-presences/src/main/java/fr/openent/statistics_presences/bean/global/GlobalSearch.java

```diff
@@ -551,8 +546,12 @@ private JsonObject groupByCountId() {
                 .put(COUNTID, String.format("$%s", COUNTID));
 
         JsonObject group = id(id)
-                .put(Field.SLOTS, sum(atLeastOne(new JsonObject().put(MongoField.$SIZE, MongoField.$ + Field.SLOTS))));
-
+                .put(Field.SLOTS, sum(atLeastOne(new JsonObject().put(MongoField.$SIZE,
```

Comment:
Permet d'éviter erreur mongo `The argument to $size must be an array, but was of type: missing`, quand un objet n'a pas de champ `slots`.